authz/rbac_translator.go

```diff
@@ -307,7 +307,7 @@ func (options *auditLoggingOptions) toProtos() (allow *v3rbacpb.RBAC_AuditLoggin
 		if config.Config == nil {
 			return nil, nil, fmt.Errorf("AuditLogger Config field cannot be nil")
```

Comment:
IIRC, empty audit logger config is a valid thing but it means that Audit is disabled

Reply:
We might've changed, but I believe last we discussed that custom config on the individual Audit Logger needed to be an empty JSON object?

Reply:
If nil means `config` field is missing from the given json, then it will be fine. We want the config to be a valid json object only if it's present.